packages/charts/examples/components/demo/advanced-usage/legend/legend-metadata/legend-metadata-example.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="nui-chart-layout">
+    <div class="chart">
+        <nui-chart class="w-100" [chart]="chart"></nui-chart>
+    </div>
+    <div class="legend">
+        <nui-legend seriesUnitLabel="Mbps" i18n-seriesUnitLabel
+                    [active]="chartAssist.isLegendActive"
+                    [interactive]="true"
+                    (mouseleave)="chartAssist.resetVisibleSeries()">
+            <!-- This is the metadata series -->
+            <nui-legend-series [interactive]="false"
+                               [descriptionPrimary]="avgSeries['name']">
+                <!-- Specify 'undefined' for the formatter key since we're not applying any special formatting to the data values shown in the legend -->
+                <!-- Specify the 'y1' data accessor to let the chart assist know how to access the data for the legend. -->
```

Comment:
Now that we've simplified the usage of getHighlightedValue here, it seems like we should move the comments back to line 27.